src/spikeinterface/sortingcomponents/matching/main.py

```diff
@@ -41,6 +51,13 @@ def find_spikes_from_templates(
 
     job_kwargs = fix_job_kwargs(job_kwargs)
 
+    if gather_mode == "memory":
+        gather_func = GatherToMemory()
+    elif gather_mode == "npy":
+        gather_func = GatherToNpy(folder, names, **gather_kwargs)
+    else:
+        raise ValueError(f"wrong gather_mode : {gather_mode}")
```

Comment:
```suggestion
        raise ValueError(f"wrong gather_mode : {gather_mode}. Please choose either 'memory' or 'npy'")
```

I think adding in useful info in cases of mistakes would be nice :)